distribution/docs/src/main/resources/_contents/_running/monitoring-contents.adoc

```diff
@@ -0,0 +1,140 @@
+
+The ${branding} contains many tools to monitor system functionality, usage, and overall system health.
+
+==== Metrics Reporting
+
+Metrics are available in several formats and levels of detail.
+
+Complete the following procedure now that several queries have been executed.
+
+. Select *${branding}-Platform*
+. Select *Metrics* tab
+. For individual metrics, choose the format desired from the desired timeframe column
```

Comment:
If choosing a format desired from desired timeframe column introduces a bulleted list there should be a colon after column